lib/vacuum.rb

```diff
@@ -0,0 +1,53 @@
+class Vacuum
```

Comment:
I had intentionally picked a general name (h/t feature of postgresql) as a placeholder for clean up of various flavors. I imagine a future where:

```
vacuum = Vacuum.new(Time.zone.parse('2016-09-30'))
vacuum.old_users
vacuum.old_proposals
```
is a regular maintenance task of some kind.

I guess we could make a `CleanerOfExpiredRecords` class instead?